### PR TITLE
Sort elements in SPDX document

### PR DESCRIPTION
**Pull request by grouigrokon**
In order to ease reading a diff between two versions of an SPDX document, both the tag:value and JSON formats of the document are now sorted.

### CODE REVIEW

File: src/e3/spdx.py [hunk 1]
```diff
@@ -957,12 +962,41 @@ def add_section(section: str) -> None:
...
+                pkg_ids.append(rel.related_spdx_element)
+            else:
+                relas.append(rel.to_tagvalue())
+
+        output += relas
+
+        packages: list[str] = ["", ""]
+
+        for pkg in sorted(self.packages.values(), key=lambda package: package.name):
+            print(f"Package ID: {pkg.spdx_id}, Document ID: {self.spdx_id}")
```
Comment (enzbang) [inline src/e3/spdx.py:990]: Is this print statement expected?

File: src/e3/spdx.py [hunk 2]
```diff
@@ -957,12 +962,41 @@ def add_section(section: str) -> None:
...
+            if pkg.spdx_id in pkg_ids:
+                print("Adding main package")
```
Comment (enzbang) [inline src/e3/spdx.py:992]: Same here

File: src/e3/spdx.py (hunk 1 above)
Comment (grouigrokon) [reply src/e3/spdx.py:990]: :'( In the middle of the stage :'( Sorry ...

File: src/e3/spdx.py (hunk 2 above)
Comment (grouigrokon) [reply src/e3/spdx.py:992]: Yes :) Why did I miss it again ... my bad :'(

File: src/e3/spdx.py (hunk 1 above)
Comment (grouigrokon) [reply src/e3/spdx.py]: Sorry, due to the presence of the `platform` package in `e3`, it became impossible to debug `e3` with PyCharm. So I added logs.\
I have to dig the configuration to make it possible (again) to debug `e3` ...

**Review by enzbang — APPROVED** [review]

**Pull request merged**